### DISCUSSION TITLE
blog/20240203/

### DISCUSSION THREAD

**Discussion by giscus-x** [xaoxuu/blog-comments#44] [Announcements]
# blog/20240203/

在回访用户的过程中，我发现有不少用户用 wiki 当作专栏来使用，可见专栏确实是一个普遍的需求点，而现有的 wiki 系统并不是专门为此场景设计的，虽然能用，但是不够好用。

https://xaoxuu.com/blog/20240203/

**Comment by weekdaycare** [comment]
支持大佬🎉如果说把博客比作一本书的话，专栏就是把书中关联性较强的几篇拿出来再做一个目录，而wiki则更像是这本书附带的一个产品说明书

**Reply by xaoxuu** [reply]
wiki 这样比喻不太合适，wiki 是一个图书馆，每一个项目是一本书，可以有 N 多个，它们之间是没有关联的。

**Comment by Whbbit1999** [comment]
好用

**Reply by xaoxuu** [reply]
快帮忙分享起来 😂

**Comment by easyf12** [comment]
支持，马上就整一个。

**Reply by xaoxuu** [reply]
这个用起来比创建和维护 wiki 项目要简单不少。

**Comment by rebron1900** [comment]
Ghost的理念也是这样

**Reply by xaoxuu** [reply]
那看来我做的是对的 😀

**Comment by Sumorio** [comment]
看完更懂这个板块的用心了。用上！

**Reply by xaoxuu** [reply]
🫶

**Comment by inkss** [comment]
左边侧栏里的导航和文章上面的导航总觉得会弄不清，有时候关注点在侧栏，就遗忘了文章上面还有一些链接了

**Reply by xaoxuu** [reply]
文章上面的你指的是标题上方的面包屑导航吗？

**Reply by inkss** [reply]
是呀是呀

**Reply by xaoxuu** [reply]
面包屑导航我觉得更偏向于提供路径信息，让读者知道自己当前看的文章属于哪个功能下，可以沿着面包屑往回走。

**Comment by aRanHC** [comment]
你好作者大大，感谢做出这么好看的主题和这么详尽的文档，请问专栏的封面图可以自定义吗？
目前都是一个打开的书本的样子。
![Snipaste_2024-04-25_18-53-30](https://aran-bucket1.oss-cn-beijing.aliyuncs.com/img_forMarkdown/Snipaste_2024-04-25_18-53-30.png)

**Reply by aRanHC** [reply]
抱歉打扰了，发现设置icon就行

**Comment by fenychn0206** [comment]
这也是我选择了stellar的原因

**Reply by xaoxuu** [reply]
🤝

**Comment by childeyouyu** [comment]
今天写博客遇到两个问题，一个是文章中间的图片没有显示（图片链接放在下面）。
另外就是设置了分类，但是点击分类会跳转会博客首页。

![image](https://youyupicgo.s3.bitiful.net/2024/11/40363653485429c1f64be0ad72d3b59b.png)​

**Comment by holwell** [comment]
我也有相关的思考，不过我用的是 halo 博客框架，和普通的博客一样官方都做了文章分类和标签字段，如果要额外添加专栏字段代价是比较大的，比较简单的实现还是利用标签，但是得将专栏标签与普通标签做区分，例如可以使用 `_专栏名` 的形式命名，之后在标签列表的展示页过滤隐藏专栏名，在专栏列表页过滤仅显示专栏名标签，在文章页的标签处对专栏标签进行独特渲染。
